compiler/prelude/prelude.go

```diff
@@ -2,9 +2,10 @@ package prelude
 
 import (
 	_ "embed"
-)
+	"log"
```

Comment:
Elsewhere we've been using logrus, which supports verbosity levels: https://github.com/gopherjs/gopherjs/blob/d0aab331249583912023f32b44e5965c9ff4ee87/build/cache/cache.go#L14